helm/kubernetes-scanner/values.yaml

```diff
@@ -207,10 +217,8 @@ metrics:
 
 resources:
   {}
-  # We usually recommend not to specify default resources and to leave this as a conscious
-  # choice for the user. This also increases chances charts run on environments with little
-  # resources, such as Minikube. If you do want to specify resources, uncomment the following
-  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # Example resources:
```

Comment:
Totally subjective, but I think we were doing a bit too much re-explanation of Kubernetes here.

Reply:
That's actually coming from the `helm init` command, not from us 😄

Reply:
Oh! TIL